src/Mvc/samples/MvcSandbox/Startup.cs

```diff
@@ -66,7 +66,8 @@ public void Configure(IApplicationBuilder app)
                         return Task.CompletedTask;
                     });
 
-                builder.MapApplication();
+                builder.MapControllers();
+                builder.MapRazorPages();
```

Comment:
We landed on them being separate instead of one MapMvc? 

#outoftheloop

Reply:
YEAH BRO.

If we need to add a *body* gesture we can do that later easily